AnkiDroid/src/main/java/com/ichi2/anki/reviewer/ActionButtonStatus.java

```diff
@@ -66,11 +66,15 @@ public void setup(SharedPreferences preferences) {
         setupButton(preferences, R.id.action_toggle_whiteboard, "customButtonEnableWhiteboard", SHOW_AS_ACTION_NEVER);
         setupButton(preferences, R.id.action_save_whiteboard, "customButtonSaveWhiteboard", SHOW_AS_ACTION_NEVER);
         setupButton(preferences, R.id.action_change_whiteboard_pen_color, "customButtonWhiteboardPenColor", SHOW_AS_ACTION_IF_ROOM);
-        setupButton(preferences, R.id.action_search_dictionary, "customButtonLookup", Lookup.isAvailable() ? SHOW_AS_ACTION_IF_ROOM : MENU_DISABLED);
+        setupButton(preferences, R.id.action_search_dictionary, "customButtonLookup", SHOW_AS_ACTION_NEVER);
     }
 
 
     private void setupButton(SharedPreferences preferences, @IdRes int resourceId, String preferenceName, int showAsActionType) {
+        if(resourceId==R.id.action_search_dictionary && !Lookup.isAvailable()){
+            mCustomButtons.put(resourceId, MENU_DISABLED);
+        }
```

Comment:
move this to the caller, it's better to explicitly special-case logic
```suggestion
        if (resourceId == R.id.action_search_dictionary && !Lookup.isAvailable()) {
            mCustomButtons.put(resourceId, MENU_DISABLED);
        }
```